src/wp-includes/rest-api/endpoints/class-wp-rest-templates-controller.php

```diff
@@ -926,6 +940,13 @@ public function get_item_schema() {
 					'type'        => 'integer',
 					'context'     => array( 'view', 'edit', 'embed' ),
 				),
+				'modified'       => array(
+					'description' => __( "The date the post was last modified, in the site's timezone." ),
```

Comment:
Tiny nit: should this refer to template?

```suggestion
					'description' => __( "The date the template was last modified, in the site's timezone." ),
```